README.md

```diff
@@ -106,6 +106,21 @@ To install the plugin in the LLDB system plugin directory, use the
 npm copy `node_modules/llnode/llnode.so` to
 `/usr/lib/lldb/plugins`.
 
+To use llnode with a core dump the core dump needs to be loaded into lldb along with the exact executable that created the core dump. The executable contains information that lldb and llnode need to make sense of the data in the core dump.
+
+To load the core dump when starting lldb use:
+```
+> lldb ~/work/node-sdks/node-v6.5.0-linux-x64/bin/node -c ~/work/core
```

Comment:
I've updated the paths and wrapped the lines.
Looking at the rest of the file $ isn't used for a prompt else where commands are left bare with no prompt character so I've gone with that convention.

I took a look at a few other projects and couldn't see a clear preference. One advantage here is that the longer multiline installation commands in this README.md use inline comments with # and those whole sections can just be pasted into a shell to do the complete installation.